hs/src/Reach/Compiler.hs

```diff
@@ -43,41 +43,31 @@ all_connectors =
       ]
 
 compile :: CompilerOpts -> IO ()
-compile copts = do
-  let outn = output copts
-  let outnMay outn_ = case intermediateFiles copts of
-        True -> Just outn_
-        False -> Nothing
+compile CompilerOpts {output, dirDotReach, intermediateFiles, installPkgs, source, tops} = do
```

Comment:
seemed easier to trace where these variables were coming from if unpacked here

Reply:
I'd rather change it to `(CompilerOpts {..})` if you're going to do that. (Note: I always parenthesize a record splat like this because I think it is confusing that one thing has a space in it

Reply:
Yeah, I'm way more into `{..}` then manually applying the field accessors :+1: especially since the data structure is defined within this file anyway